packages/expo-router/src/modal/web/TransparentModalStackRouteDrawer.web.tsx

```diff
@@ -0,0 +1,71 @@
+'use client';
+import React from 'react';
+import { Drawer } from 'vaul';
+
+import modalStyles from './modalStyles';
+import { ExtendedStackNavigationOptions } from '../../layouts/StackClient';
+
+function TransparentModalStackRouteDrawer({
+  routeKey,
+  options,
+  renderScreen,
+  onDismiss,
+}: {
+  routeKey: string;
+  options: ExtendedStackNavigationOptions;
+  renderScreen: () => React.ReactNode;
+  onDismiss: () => void;
+}) {
+  const handleOpenChange = (open: boolean) => {
+    if (!open) onDismiss();
+  };
+
+  return (
+    <Drawer.Root
+      defaultOpen
+      key={`${routeKey}-transparent`}
+      dismissible={options.gestureEnabled ?? true}
```

Comment:
I don't think we should support gesture dismiss in this case. At least this is not supported on iOS in case of full screen modal

Reply:
I was thinking the same, in my local branch I turned the default to false, leaving the option to enable it?

Reply:
I allow an overwrite but default is false now